plans/openssl/plan.sh

```diff
@@ -39,6 +40,8 @@ do_prepare() {
 }
 
 do_build() {
+  # Set PERL var for scripts in `do_check` that use Perl
+  export PERL=$(pkg_path_for core/perl)/bin/perl
```

Comment:
argh I expanded it but apparently you need to expand twice. Remember kids: https://caragaleblog.files.wordpress.com/2015/10/zombieland-2.gif